### PR TITLE
Transaction payment runtime apis: query call info and fee details

### PR DESCRIPTION
**Pull request by muharem**
Fixes https://github.com/paritytech/substrate/issues/11665

New runtime api introduced in PR https://github.com/paritytech/substrate/pull/11819

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -451,7 +451,7 @@ dependencies = [
 [[package]]
 name = "beefy-gadget"
 version = "4.0.0-dev"
-source = "git+https://github.com/paritytech/substrate?branch=master#fece0657f20e15df94be5833b164dfacd44823eb"
```
Comment (muharem) [inline Cargo.lock:454]: result of `cargo update -p pallet-transaction-payment-rpc-runtime-api`

**Review by athei — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Comment by muharem** [comment]
`continuous-integration/gitlab-check-runtime` check fail can be ignored, explained - https://github.com/paritytech/ci_cd/issues/538

**Review by ggwpez — APPROVED** [review]
I don't see any RPC integration tests which query a node, maybe we should add that in the future.

**Review by dvdplm — APPROVED** [review]

**Comment by muharem** [comment]
bot merge

**Pull request merged**